pkg/kubelet/network/dns/dns_test.go

```diff
@@ -247,10 +247,10 @@ func TestMergeDNSOptions(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
-		options := mergeDNSOptions(tc.existingDNSConfigOptions, tc.dnsConfigOptions)
+		options := MergeDNSOptions(tc.existingDNSConfigOptions, tc.dnsConfigOptions)
```

Comment:
Can we do the test without referencing the functions that are used to implement the feature? This seems a bit circular dep (suppose MergeDNSOptions has a bug...).

Reply:
Indeed, will look into removing that dependency.